db/migrate/20190212111202_create_policies.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePolicies < ActiveRecord::Migration
+  def change
+    create_table :policies do |t|
+      t.string :name
```

Comment:
For all those it should be `null: false`

Reply:
@hallelujah I am currently validating the presence in the model:
https://github.com/3scale/porta/pull/593/commits/eb0bb580ab9a365ce666a87e0781faf9eb7ab7bd
But I can ass the `null: false` in the DB as well, ok 👍

Reply:
@hallelujah I am currently validating the presence in the model:
https://github.com/3scale/porta/pull/593/commits/eb0bb580ab9a365ce666a87e0781faf9eb7ab7bd
But I can add the `null: false` in the DB as well, ok 👍

Reply:
I am not sure if it is worth it though, let me know after you check that commit please @hallelujah

Reply:
Added